tests/test_point.py

```diff
@@ -5,6 +5,61 @@
 import pytest
 
 
+def test_from_coordinates():
+    # 2D points
+    p = Point(1.0, 2.0)
+    assert p.coords[:] == [(1.0, 2.0)]
+    assert p.has_z is False
+
+    # 3D Point
+    p = Point(1.0, 2.0, 3.0)
+    assert p.coords[:] == [(1.0, 2.0, 3.0)]
+
+    # empty
+    p = Point()
+    assert p.is_empty
+    assert p.coords[:] == []
+
+
+def test_from_sequence():
+    # From single coordinate pair
+    p = Point((3.0, 4.0))
+    assert p.coords[:] == [(3.0, 4.0)]
+    p = Point([3.0, 4.0])
+    assert p.coords[:] == [(3.0, 4.0)]
+
+    # From coordinate sequence
+    p = Point([(3.0, 4.0)])
+    assert p.coords[:] == [(3.0, 4.0)]
+
```

Comment:
I wonder if 3D sequences should be added here too (and in the following four tests too).

Reply:
Added